src/Network/Network/Generated/Models/PSExpressRouteCircuit.cs

```diff
@@ -62,6 +62,8 @@ public partial class PSExpressRouteCircuit : PSTopLevelResource
         public int? Stag { get; set; }
         [Ps1Xml(Target = ViewControl.Table)]
         public string AuthorizationKey { get; set; }
+        [Ps1Xml(Target = ViewControl.Table)]
+        public string AuthorizationStatus { get; set; }
```

Comment:
How do you set value for AuthorizationStatus?

Reply:
@BethanyZhou its a read-only property

Reply:
```suggestion
        public string AuthorizationStatus { get; }
```
I mean how do you pass the value from SDK to this property